modules/installation-uninstall-clouds.adoc

```diff
@@ -147,13 +147,6 @@ $ ccoctl ibmcloud delete-service-id \
     --credentials-requests-dir <path_to_credential_requests_directory> \
     --name <cluster_name>
 ----
-+
---
-[NOTE]
-====
-If your cluster uses Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set, you must include the `--enable-tech-preview` parameter.
```

Comment:
This pull request currently seems unsettled about whether `ccoctl` will need `--enable-tech-preview`.  We probably need to keep it everywhere for now, and in fact, we can unconditionally recommend its use everywhere.  For this example:

```console
$ ccoctl ibmcloud delete-service-id \
    --credentials-requests-dir <path_to_credential_requests_directory> \
    --enable-tech-preview \
    --name <cluster_name>
```

And then we can file `ccoctl` tickets to have:

* 4.15 or some such pivot to "`--enable-tech-preview` is enabled by default", so it can be dropped from docs.
* 4.16 or some such pivot to "`--enable-tech-preview` is deprecated", so folks who have scripted out the old approach will hear that it's no longer doing anything useful.
* 4.30 or some such pivot to "`--enable-tech-preview` is no longer supported", to clear out some cruft that hopefully nobody is consuming anymore.

And maybe that dance is enough reason to leave "you may need to include `--enable-tech-preview`..." in a `NOTE` for now, until we get to the "enabled by default" stage?

Reply:
Updated to add the `--enable-tech-preview` notes back where they had been removed. So we are at least starting from a consistent place and can go from there however makes sense :slightly_smiling_face: